cumulusci/tasks/github/publish.py

```diff
@@ -45,21 +49,60 @@ def _init_options(self, kwargs):
                 "include", ["datasets/", "documentation/", "tasks/", "unpackaged/"]
             )
         )
+
+        self.options["renames"] = self._process_renames(self.options.get("renames", []))
+
         if self.options.get("version") in ("latest", "latest_beta"):
             get_beta = self.options["version"] == "latest_beta"
             self.options["version"] = str(
                 self.project_config.get_latest_version(beta=get_beta)
             )
         if "ref" not in self.options and "version" not in self.options:
             raise TaskOptionsError("Either `ref` or `version` option is required")
+
         self.options["create_release"] = process_bool_arg(
-            self.options.get("create_release", True)
+            self.options.get("create_release", False)
         )
         self.options["release_body"] = process_bool_arg(
             self.options.get("release_body", False)
         )
         self.options["dry_run"] = process_bool_arg(self.options.get("dry_run", False))
 
+    def _process_renames(self, renamed_paths):
+        """
+        For each entry in renames, any renames and store them
+        in self.local_to_target_paths.
+        """
+        is_list_of_dicts = all(
+            [isinstance(pair, dict) for pair in renamed_paths if renamed_paths]
+        )
```

Comment:
Can't this simply be:
```suggestion
        is_list_of_dicts = all(isinstance(pair, dict) for pair in renamed_paths)
```
?

Reply:
I always forget about generator expressions, thanks.